packages/ChangesReloaded-Tests.package/CRFakeFileStream.class/instance/basicUpTo..st

```diff
@@ -3,6 +3,6 @@ basicUpTo: aDelimiter
 
 	| returnString |
 	returnString := ''.
-	[(self position < self string size) and: (self peek ~~ aDelimiter)] whileTrue: [ returnString := returnString , self next].
+	[(self position < self string size) and: (self peek ~~ aDelimiter)] whileTrue: [returnString := returnString , self next].
```

Comment:
```suggestion
	[(self position < self string size) and: (self peek ~~ aDelimiter)] 
		whileTrue: [returnString := returnString , self next].
```
didn't we want to do it like this so that the line would be a bit shorter?

Reply:
we are still very insecure about the whole new line business, we only talked explicitly about another case (which kinda shows how arbitrary this whole new line thing is because it wasn't a problem there until now) - but I think that would be ok :)

Reply:
I like the newline here too